index.html

```diff
@@ -656,6 +664,10 @@ <h4>Whitespace Normalization</h4>
 	    
 	    <section id="accents">
 			<h4>Accents and diacritic marks</h4>
+
+			<p class="advisement">By default, string searching SHOULD ignore diacritics.</p>
+
+			<p class="advisement">User agent MAY provide an option for diacritics-sensitive search where precision is critical.</p>
```

Comment:
```suggestion
			<p class="advisement">User agents SHOULD provide an option for diacritics-sensitive search.</p>
```

Reply:
Personally, I tend to keep this as a MAY. I know this feature is useful for many people, but some user agents might want a minimalist UI and not expose too many options to users. In this case, they might want to offer a default search method, and SHOULD might be too much for them.